ksp_plugin_adapter/flight_planner.cs

```diff
@@ -128,7 +128,7 @@ class FlightPlanner : SupervisedWindowRenderer {
     }
 
     if (burn_editors_ != null) {
-      string vessel_guid = vessel_?.id.ToString();
+      string vessel_guid = vessel_!.id.ToString();
```

Comment:
Can we have a comment explaining why `vessel_` cannot be null here?

I find this function very confusing in general, perhaps it should be split in a future pull request.

Reply:
Added first order logic annotations.